src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,6 +309,16 @@ export class RunParametersAction extends TooltipFromLabelAction {
 	*/
 	public async run(context: URI): Promise<void> {
 		const editor = this._notebookService.findNotebookEditor(context);
+		// Only run action for kernels that are supported (Python, PySpark, PowerShell)
+		let kernelsNotSupported = ['sql', 'kusto'];
+		if (kernelsNotSupported.includes(editor.model.language)) {
+			// If there is no parameters in the cell indicate to user to add them
+			this.notificationService.notify({
```

Comment:
It would be nice to have separate messages that indicated exactly what the problem was - there's no indication to them that the reason they can't run is because they have the wrong kernel.